### ISSUE TITLE
[Issue]: Data lost after update from 10.8.4 to 10.8.5 using windows installer.

### ISSUE THREAD

**Issue by MediumME** [jellyfin/jellyfin#8454]
### Please describe your bug

To update my server from version 10.8.4 to 10.8.5 on windows, i did as the install docu suggest for updating. The installer prompt says that i would not touch any settings etc., but unfortunately my data (metadata, plugins, libraries etc.) has been wiped during the update. No logging in afterwards through whatever method. Had to setup a new server.

Installer used:
[installer/jellyfin_10.8.5_windows-x64.exe](https://repo.jellyfin.org/releases/server/windows/stable/installer/jellyfin_10.8.5_windows-x64.exe)

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS: Windows 11
- Virtualization:
- Clients: Browser, Android
- Browser: Brave v1.43.93
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins: 
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nielsvanvelzen** [comment]
Did you close Jellyfin before starting the update? Did the installer show an alert that it would update an existing installation? How do you start Jellyfin, with the tray app or directly? Did you install Jellyfin as a service?

**Comment by MediumME** [comment]
Yes, stopped jellyfin and closed the tray icon before updating. The installer showed said alert. Usually Jellyfin is autostarting with windows, after updating i launched the tray app to start jellyfin. I did not install jellyfin as a service.

**Comment by ThomasPGH** [comment]
Are you sure the installation has wiped your Jellyfin data folder?
Maybe it's just on a different drive/in a different folder, and after the update some other drive/folder was used.

I keep having this issue every time I update Jellyfin ever since I updated it for the first time (maybe 10. something).

When starting `nssm edit "Jellyfin Server"` I got `--service --datadir "D:\ProgramData\Jellyfin\Server"` in the Arguments field because this is where I want Jellyfin to keep its configs and metadatay.

However, every time I run an update to a newer version of Jellyfin it resets this text field to some odd `--service --datadir "M:\ProgramData\Jellyfin\Server"`, although this folder doesn't even exist. I have to manually change it back to how it was before the update.

Maybe it has done the same when you updated but the folder it picked did exist, hence the "data wipe".
